common/logging_cli.go

```diff
@@ -0,0 +1,127 @@
+package common
+
+import (
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/urfave/cli/v2"
+)
+
+/*
+	TODO: https://github.com/Layr-Labs/eigenda-proxy/issues/268
+
+	This CLI logic is already defined in the eigenda monorepo:
+	 https://github.com/Layr-Labs/eigenda/blob/0d293cc031987c43f653535732c6e1f1fa65a0b2/common/logger_config.go
+
+	This regression is due to the fact the proxy leverage urfave/cli/v2 whereas
+	core eigenda predominantly uses urfave/cli (i.e, v1).
+
+*/
+
+const (
+	PathFlagName   = "path"
+	LevelFlagName  = "level"
+	FormatFlagName = "format"
+	// deprecated
+	PidFlagName   = "pid"
+	ColorFlagName = "color"
+)
+
+type LogFormat string
+
+const (
+	JSONLogFormat LogFormat = "json"
+	TextLogFormat LogFormat = "text"
+)
+
+type LoggerConfig struct {
+	Format       LogFormat
+	OutputWriter io.Writer
+	HandlerOpts  logging.SLoggerOptions
+}
+
+func LoggerCLIFlags(envPrefix string, flagPrefix string) []cli.Flag {
+	category := "logging"
+
+	return []cli.Flag{
+		&cli.StringFlag{
+			Name:     common.PrefixFlag(flagPrefix, LevelFlagName),
+			Category: category,
+			Usage:    `The lowest log level that will be output. Accepted options are "debug", "info", "warn", "error"`,
+			Value:    "info",
+			EnvVars:  []string{common.PrefixEnvVar(envPrefix, "LOG_LEVEL")},
+		},
+		&cli.StringFlag{
+			Name:     common.PrefixFlag(flagPrefix, PathFlagName),
+			Category: category,
+			Usage:    "Path to file where logs will be written",
+			Value:    "",
+			EnvVars:  []string{common.PrefixEnvVar(envPrefix, "LOG_PATH")},
+		},
```

Comment:
I would get rid of this, as explained in my other comment

Reply:
would prefer if we kept it since this entire artifact is copy pasta from upstream